repos/system_upgrade/el7toel8/actors/powertop/actor.py

```diff
@@ -0,0 +1,32 @@
+from leapp.actors import Actor
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.reporting import Report
+from leapp.libraries.common.reporting import report_with_remediation
+
+
+class PowerTop(Actor):
+    """
+    Check if PowerTOP is installed. If yes, write information about non-compatible changes.
+    """
+
+    name = 'powertop'
+    consumes = (InstalledRedHatSignedRPM,)
+    produces = (Report,)
+    tags = (ChecksPhaseTag, IPUWorkflowTag)
+
+    def process(self):
+        for fact in self.consume(InstalledRedHatSignedRPM):
+            for rpm in fact.items:
+                print(rpm)
```

Comment:
Is this "print" really necessary? If so, could we use a debug message from our log system?

Reply:
Ah, sorry, I forgot it there :) It was used for debugging.